### ISSUE TITLE
Sample size training dataset

### ISSUE THREAD

**Issue by vis6060** [WuJunde/MedSegDiff#123]
For this particular model mentioned in the paper, what is the minimum training sample size dataset to get a prediction of the segmentation masks?  Is it possible that 100 images for the training dataset, trained for 100K steps and all the parameters mentioned in the suggested python command is sufficient?

**Comment by WuJunde** [comment]
no, I think at least 1000 to get a fine result
